src/robot.ts

```diff
@@ -1,4 +1,4 @@
-export type { Robot } from './robot/robot';
+export type { Robot, RobotStatusStream } from './robot/robot';
```

Comment:
we should export this in `src/main.ts` as well so users have access to this type